src/chart/sankey/SankeyView.ts

```diff
@@ -220,6 +220,16 @@ class SankeyView extends ChartView {
                     }
             }
 
+            setLabelStyle(
```

Comment:
I suggest fixing the position by setting `curve.setTextConfig({ position: 'inside' })` after `setLabelStyle` instead of adding a new config to the function if it's not common.

If we want to providing the ability to limit the positions in the `setLabelStyle`. A more general way is using an array and check if user-given position is in this array. We can name it availablePositions.